stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -50,6 +50,16 @@ interface StripeIntent : StripeModel {
 
     val nextActionData: NextActionData?
 
+    /**
+     * Confirmation has succeeded and all required actions have been handled.
+     */
+    val isConfirmed: Boolean
+        get() = setOf(
+            Status.Processing,
+            Status.RequiresCapture,
+            Status.Succeeded
+        ).contains(status)
```

Comment:
FYI SetupIntents don't have a `RequiresCapture` status: 

https://stripe.com/docs/api/setup_intents/object#setup_intent_object-status

Reply:
Sure, but this logic is still valid to both PI and SI.

Reply:
I have tried to audit this code in the past, and having states that apply to one but not the other gets really confusing.  It might be better if each setup intent and payment intent implements this method with only the states that are valid for each?

Reply:
Done.